cli/commands.js

```diff
@@ -159,6 +159,10 @@ Example:
     openshift: {
         desc: 'Deploy the current application to OpenShift',
     },
+    page: {
```

Comment:
Yes for now, it's vue specific: it's for creating a new page.
What could be done:
- rename to vue-page
- add this little feature to Angular and React too, so it will help to create a new page easily